conf/hbase-env.sh

```diff
@@ -143,10 +143,32 @@
 # export GREP="${GREP-grep}"
 # export SED="${SED-sed}"
 
-# Uncomment to enable trace, you can change the options to use other exporters such as jaeger or
-# zipkin. See https://github.com/open-telemetry/opentelemetry-java-instrumentation on how to
+# Tracing
+# Uncomment some combination of these lines to enable tracing. You should change the options to use
+# the exporters appropriate to your environment. See
+# https://github.com/open-telemetry/opentelemetry-java-instrumentation for details on how to
 # configure exporters and other components through system properties.
-# export HBASE_TRACE_OPTS="-Dotel.resource.attributes=service.name=HBase -Dotel.traces.exporter=logging -Dotel.metrics.exporter=none"
+#
+# The presence HBASE_TRACE_OPTS indicates that tracing should be enabled, and serves as site-wide
+# settings.
+# export HBASE_TRACE_OPTS="-Dotel.traces.exporter=none -Dotel.metrics.exporter=none"
```

Comment:
I don't follow your note here, @taklwu . Are you adding here a reference to the upstream documentation? Was there something else you need?

Reply:
my original thought was the instruction for exporters wasn't straightforward that we link to [open-telemetry-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation) instead of another git repo/link under [opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md), e.g. here it's `otel.traces.exporter`.  

so I just put it as a note in this PR for other people search online (or if you find this link helpful, we can add it to the code here), such that operators know we have many options for `otel.traces.exporter` including `logging`, `otlp`, `none`, `jaeger`, `zipkin`. 

```
# You should change the options to use
# the exporters appropriate to your environment. See
# https://github.com/open-telemetry/opentelemetry-java-instrumentation for details on how to
# configure exporters and other components through system properties.
```